Cineaste/CoreDataHelper/MovieStorageManager.swift

```diff
@@ -11,19 +11,22 @@ import CoreData
 
 class MovieStorageManager {
     let persistentContainer: NSPersistentContainer
+    let backgroundContext: NSManagedObjectContext
 
     // MARK: Init with dependency
-    init(container: NSPersistentContainer = AppDelegate.persistentContainer) {
-        self.persistentContainer = container
-        self.persistentContainer.viewContext.automaticallyMergesChangesFromParent = true
+    init(container: NSPersistentContainer = AppDelegate.persistentContainer, useViewContext: Bool = false) {
+        persistentContainer = container
+        persistentContainer.viewContext.automaticallyMergesChangesFromParent = true
+
+        if useViewContext {
```

Comment:
Is there a better solution than this? We only need this for the unit tests, because they need to use the viewContext.

Reply:
A simple solution might be to always use the view context.

> “As a general rule, you should use the simplest stack you can get away with.”

-- Excerpt From: Florian Kugler. “Core Data”. Apple Books.

Reply:
I'm looking into other workarounds.

Reply:
Another workaround could be [this][1]: Wait in the tests for the `viewContext.mergeChanges(fromContextDidSave:)` to happen.



[1]: https://github.com/xavierLowmiller/cineaste-ios/commit/1f87a9451668b49af8c2d0d395875031fe33b4a5